doc/man3/SSL_CTX_set_new_pending_conn_cb.pod

```diff
@@ -0,0 +1,48 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_new_pending_conn_cb, SSL_set_new_pending_conn_cb_fn - callback function to report creation of QUIC connection SSL objects 
+
+=head1 SYNOPSIS
+
+ typedef int (*SSL_set_new_pending_conn_cb_fn)(SSL_CTX *c, SSL *new_ssl,
+                                               void *arg);
+ void SSL_CTX_set_new_pending_conn_cb(SSL_CTX *c,
+                                     SSL_set_new_pending_conn_cb_fn *f,
+                                     void *arg);
+
+=head1 DESCRIPTION
+
+SSL_CTX_set_new_pending_conn_cb() sets the new_pending_conn callback function and
+associated application data argument I<arg>.  When using the QUIC transport, TLS
+handshake processing may occur independently from the thread which accepts the
+connection that the handshake is establishing.  As such, B<SSL> objects
+representing the connection may be allocated and initialized prior to a call to
+SSL_accept_connection().
```

Comment:
This description seems to be missing the punchline. The context is described, but not the purpose of the callback, which is to allow the application to record the association between the new object and the parent SSL handle, perhaps adding ex_data to the new object.